src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -66,6 +71,7 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
     props: Props;
 
     state: State = {
+        isArchiveFile: false,
```

Comment:
Minor, but consider `isArchived` to stay consistent with `isWatermarked`.

Reply:
Updated